.circleci/parallel_bytecode_report.sh

```diff
@@ -44,6 +44,9 @@ cd test-cases/
 echo "Preparing input files"
 python3 ../scripts/isolate_tests.py ../test/
 
+# FIXME: These cases crash because of https://github.com/ethereum/solidity/issues/13583
+rm ./*_bytecode_too_large_*.sol ./*_combined_too_large_*.sol
```

Comment:
@ekpyron dropped #13496 from the focus board recently so I guess he does not want use to focus on fixing in this. For that reason it'll stay like this for the time being. I.e. we're not really running the comparison on **all** test cases from the repo.